src/components/virtualRepeater/virtualRepeater.js

```diff
@@ -0,0 +1,452 @@
+/**
+ * @ngdoc module
+ * @name material.components.virtualRepeat
```

Comment:
I think this is a doc bug. Should be `@name material.components.virtualRepeater`